.travis.yml

```diff
@@ -0,0 +1,27 @@
+dist: jammy
+language: node_js
+node_js:
+  - lts/*
```

Comment:
Should this match the `.node-version` more precisely?

Travis docs say they _do_ support using a `.nvmrc` file directly, but not `.node-verson`, I guess.

Reply:
Talked in person, but yeah my thoughts are I think lts is fine